touchforms/formplayer/static/formplayer/script/fullform-ui.js

```diff
@@ -394,22 +394,31 @@ function Question(json, parent) {
       //this.control.restore_ui_state(uistate);
     }
 
-    var add_multimedia = function(attrib, control) {
-      if (self.hasOwnProperty(attrib) && self[attrib]) {
-        var mediaSrc = getForm(self).adapter.render_context.resourceMap(self[attrib]);
-        if (mediaSrc) {
-          control.attr("src", mediaSrc);
-          $mediaContainer = $('<div>');
-          $mediaContainer.append(control);
-          var $widget = self.$container.find('#widget');
-          if ($widget.length) {
-            $widget.append($mediaContainer);
-          } else {
-            $capt.append($mediaContainer);
+    var add_multimedia = function (attrib, control) {
+      var getMediaSrc = function (attrib) {
+        if (self.hasOwnProperty(attrib) && self[attrib]) {
+          try {
+            return getForm(self).adapter.render_context.resourceMap(self[attrib]);
```

Comment:
this kind of seems like a hammer. if anything were to break in the resourceMap function we'd never know. kind of like using `except Exception`. shouldn't the change be made in the `resourceMap`? https://github.com/dimagi/commcare-hq/blob/fad41eb69d050c6e773db125fe25b85e192b2469/corehq/apps/cloudcare/static/cloudcare/js/backbone/apps.js#L703